spec/support/idv_helpers.rb

```diff
@@ -11,7 +11,7 @@ def verify_identity_with_doc_auth
     click_on 'Continue'
     expect(page).to have_current_path('/verify/doc_auth/upload')
 
-    click_on 'Use your computer'
+    click_on 'Upload from your computer'
```

Comment:
We still need to click `Upload from your computer` if it is present because that is what is in staging and prod

Reply:
Actually, this will work because we only run these tests in staging and prod